app/controllers/checkouts_controller.rb

```diff
@@ -181,5 +179,11 @@ def checkin_bak
       end
     end
   end
+
+  # def reply
```

Comment:
http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file